Main/Source/bodypart.cpp

```diff
@@ -3244,10 +3237,10 @@ void bodypart::ReceiveAcid(material* Material, cfestring& LocationName, long Mod
         if(Master->IsPlayer())
         {
           cchar* TName = LocationName.IsEmpty() ? GetBodyPartName().CStr() : LocationName.CStr();
-          ADD_MESSAGE("Acidous %s dissolves your %s.", MName.CStr(), TName);
+          ADD_MESSAGE("Caustic %s dissolves your %s.", MName.CStr(), TName);
```

Comment:
"Caustic" is a descriptor for substances that are bases (as opposed to acids, which are acidic). For example, sodium hydroxide, or NaOH, used for unclogging blocked shower drains, is known around the home as "caustic soda".

Indeed, the adjective for acids is "acidic", which could be used instead. However, "acidous" is also a nice word. It's an unobtrusive portmanteau of "acid" and "-ous". Many nice adjectives are "-ous" words, such as porous, enormous, hazardous, and prosperous, to name a few. The word "acidous" has an archaic sound and feel to it which I think suits the world of IVAN. As authors, we have the poetic license to deploy such words in our fantasy worlds. For instance, (and please forgive me my awful fantasy-writing style):

>"A conjuror arrived in our village late one night. He got so drunk at the tavern that when he staggered out-o'-doors he fell down the well and was drownded. The water's been acidous ever since." 

"Acidous" is one of those rare words with some semblance of historical plausibility to it. Maybe by using it here, we can fool people into using it in real life? Shakespeare ex Machina.

By contrast, "acidic" is the scientifically accurate word to describe an acid. Its use here would make things unambiguous for all players of IVAN.

So @red-kangaroo, I leave the choice with you. Either "acidic", or "acidous", but certainly not "caustic".

Reply:
Interesting! I thought "acidic" is for acids, "basic" for bases and "caustic" for either.

I'll revert it to acidous. The change was frankly made only because "Acidous sulphuric acid dissolves the foo." or "Acidous acidous blood dissolves the foo." read weird to me. :)

Reply:
I get it now, sounds redundant. May be
"Acidic sulphuric acid dissolves the foo."
"Acidic acidous blood dissolves the foo."
would sound less repetitive/weird in these cases, being more precise even if lacking a bit of immersion :>
The best anyway would be a check for duplicated words, but that would require extra code...

Reply:
I was wrong.

> I thought "acidic" is for acids, "basic" for bases and "caustic" for either.

@red-kangaroo you are right, "caustic" describes the ability of a substance (acid or base) to burn, corrode, dissolve, or eat away by chemical action, a solid such as flesh.

As you pointed out, we end up with messages like "Acidous acidous blood", so I think perhaps we should use "caustic" after all. That would obviate the requirement for duplicated word checking.

My dinner tonight:
:plate_with_cutlery: <- :tophat: